infra/base-images/base-builder/precompile_centipede

```diff
@@ -18,8 +18,8 @@
 echo -n "Precompiling centipede"
 
 # Build Centipede with bazel.
-cd "$SRC/centipede/"
-apt-get install libssl-dev -y
+cd "$SRC/fuzztest/centipede/"
+apt-get upgrade && apt-get install libssl-dev -y
```

Comment:
Why is the upgrade needed?

Reply:
Hmm, I am not sure.
Usually, I use `update` to ensure `apt-get` can find the package to install.